doc/Comments.xml

```diff
@@ -300,9 +300,11 @@ Resets the current level to 0.
 <Subsection Label="@BeginExample">
 <Index Key="@BeginExample"><C>@BeginExample and @EndExample</C></Index>
 <Heading>@BeginExample and @EndExample</Heading>
-@BeginExample inserts an example into the manual. The syntax is like the example environment
-in GAPDoc. This examples can be tested by GAPDoc, and also stay readable by GAP.
-The GAP prompt is added by AutoDoc.
+@BeginExample inserts an example into the manual. The syntax for examples is different from
+GAPDocs example syntax in order to have a
+file that contains the example and is GAP readable. GAP commands are not preceded by a comment, while output has to be preceded by an
+&AutoDoc; comment. So examples can be tested by GAPDoc, and also stay readable by GAP.
+The GAP prompt for the display in the manual is added by AutoDoc.
```

Comment:
I find this a bit hard to understand. Perhaps remove this, instead add: "For example, consider this: [ Example block follows]. As you can see, the commands are not commented out, and hence are executed if the file containing this example block is read. At the same time, when building the documentation, this inserts an example equivalent to what this would produce: [ insert equivalent Begin/EndExampleSesson ]"